README.md

```diff
@@ -1133,6 +1133,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
 
 ## Generative AI
 
+  * [Zenable](https://zenable.io) - Instantly auto-fix outputs from tools like Cursor, Windsurf, and Copilot to meet your company's quality and compliance standards using guardrails built with Policy as Code. The free tier includes unlimited MCP server usage and 25 free automated pull request reviews per day via the GitHub App.
```

Comment:
does the mcp server talk to a SaaS or purely local?

Reply:
@ripienaar sorry I missed your question. It's a SaaS